docs/mdoc/user-guide.md

```diff
@@ -756,7 +776,7 @@ or `beShorterThan(n: Int)`.
 
 ### Errors
 
-An endpoint may fail (it may evaluate into a `Future.exception`) by a number of reasons: it was
+An endpoint may fail (it may evaluate into a `Throwable`) by a number of reasons: it was
```

Comment:
I think it might evaluate into `F[Throwable]`

Reply:
fixed.